src/metax_api/tests/api/rest/v2/views/datasets/drafts.py

```diff
@@ -64,12 +64,32 @@ def _set_cr_owner_and_state(self, cr_id, state, owner):
         cr.data_catalog_id = DataCatalog.objects.get(catalog_json__identifier=END_USER_ALLOWED_DATA_CATALOGS[0]).id
         cr.force_save()
 
-    def test_field_exists(self):
+    def test_field_state_exists(self):
         """Try fetching any dataset, field 'state' should be returned'"""
 
         cr = self.client.get('/rest/v2/datasets/13').data
         self.assertEqual('state' in cr, True)
 
+    def _test_issued_date_is_not_generated_for_drafts(self):
+        '''  Drafts will not have the issued date generated
+             Field is created when dataset is published
+        '''
```

Comment:
nitpick: multiline comments either in a inline manner (example on lines 244-245) or in a block type manner (example on lines 26-28)